src/components/footer/Footer.scss

```diff
@@ -1,3 +1,249 @@
-footer {
+.footer {
+  position: absolute;
+  margin-bottom: 22%;
+  width: 100%;
+  height: 510px;
+  background-image: url("./../../../public/pattern\1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #080F20;
+}
+
+.footer__text-h2 {
+  color: #fff;
+  font-size: 32px;
+  width: 500px;
+  margin-top: 5%;
+  margin-left: 30%;
+  text-align: center;
+  word-spacing: 3px;
+  letter-spacing: 0.5px;
+  line-height: 45px;
+}
+
+
+.footer__text-p {
+  color: #CCD5E0;
+  font-size: 14px;
+  margin-left: 39%;
+}
+
+
+.footer__text-btn {
+  margin-left: 46%;
+  padding: 12px 22px;
+  color: #FF8D13;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  transition: 0.3s ease;
+  border-radius: 1px;
+  font-size: 14px;
+
+  &:hover {
+    background-color: #FF8D13;
+    color: #fff;
+
+  }
+}
+
+
+.footer__line {
+  border-bottom: 1px solid #424955;
   display: block;
+  margin-top: 100px;
+  width: 800px;
+  margin-left: 157px;
+}
+
+
+.footer__box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 70px;
+
+  h2 {
+    color: #fff;
+    font-size: 12px;
+    margin-bottom: 23px;
+    margin-top: 37px;
+  }
+}
+
+.footer__part-1 {
+
+  img {
+    width: 125px;
+    height: 47px;
+    margin-left: 17%;
+  }
+
+   p {
+    color: #838491;
+    font-size: 9px;
+    margin-top: -21px;
+    width: 120px;
+    margin-left: 17%;
+    line-height: 1.5;
+}
+}
+
+
+.footer__part-2 {
+
+  .footer__part2-h2{
```

Comment:
nega bir xil narsalarga xarhil stil berilgan? nega margin foizda berilgan? to'g'irlang